proxy/src/main/java/com/velocitypowered/proxy/connection/client/ConnectedPlayer.java

```diff
@@ -838,18 +838,20 @@ public void sendResourcePack(String url) {
     ResourcePackRequest request = new ResourcePackRequest();
     request.setUrl(url);
     request.setHash("");
+    request.setRequired(false);
     connection.write(request);
   }
 
   @Override
-  public void sendResourcePack(String url, byte[] hash) {
+  public void sendResourcePack(String url, byte[] hash, boolean isRequired) {
```

Comment:
Maybe also keep the previous method for compat?

Reply:
See https://github.com/VelocityPowered/Velocity/pull/382/files#diff-433700b629ed381aec737f3b64379fa3aa63f8b7d93191e73d3a79329ae8b9ebR221